dev/com.ibm.ws.logging/resources/com/ibm/ws/logging/internal/resources/LoggingMessages.nlsprops

```diff
@@ -123,6 +123,15 @@ MESSAGE_FORMAT_UPDATED.useraction=No action is required.
 CONSOLE_FORMAT_UPDATED=TRAS3003I: The console output format has now been updated to '{0}'.
 CONSOLE_FORMAT_UPDATED.explanation=The output format for console has now been updated.
 CONSOLE_FORMAT_UPDATED.useraction=No action is required.
+
+FAILED_TO_WRITE_LOG=TRAS3004E: Failed to write message to file {0}.
```

Comment:
Suggested change:
Failed to write message**s** to **the {0} file**.
or:
**Cannot** write message**s** to **the {0} file**.